### PR TITLE
Task/hw 53475 update hyperwallet transfer method

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: Sources/Model/TransferMethod/HyperwalletTransferMethod.swift [hunk 1]
```diff
@@ -133,10 +140,12 @@ public class HyperwalletTransferMethod: Codable {
     /// - bankAccount:   When the transfer method is Bank Account
     /// - bankCard:      When the transfer method is Bank Card
     /// - payPalAccount: When the transfer method is PayPal Account
+    /// - wireAccount:   When the transfer method is Wire Account
     public enum TransferMethodType: String {
         case bankAccount = "BANK_ACCOUNT"
         case bankCard = "BANK_CARD"
         case payPalAccount = "PAYPAL_ACCOUNT"
+        case wireAccount = "WIRE_ACCOUNT"
```
Comment (syyang-hw) [inline Sources/Model/TransferMethod/HyperwalletTransferMethod.swift]: Can we modify the build of HyperwalletBankAccount to this? 

```
  public init(transferMethodCountry: String,
                    transferMethodCurrency: String,
                    transferMethodProfileType: String,
                    transferMethodType: String) {
            storage[TransferMethodField.type.rawValue] = AnyCodable(value: transferMethodType)
            storage[TransferMethodField.transferMethodCountry.rawValue] = AnyCodable(value: transferMethodCountry)
            storage[TransferMethodField.transferMethodCurrency.rawValue] = AnyCodable(value: transferMethodCurrency)
            storage[TransferMethodField.profileType.rawValue] = AnyCodable(value: transferMethodProfileType)
        }
```

Comment (syyang-hw) [reply Sources/Model/TransferMethod/HyperwalletTransferMethod.swift]: Make sure all the tests pass and update all the usages of this builder

**Comment by syyang-hw** [comment]
Please make sure the code coverage not dropping.

File: Sources/Model/TransferMethod/HyperwalletTransferMethod.swift (hunk 1 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Model/TransferMethod/HyperwalletTransferMethod.swift]: @syyang-hw Requested changes provided. Tests pass, code coverage increased.

**Pull request merged**